src/main/java/com/socialmedia/controllers/UserController.java

```diff
@@ -0,0 +1,17 @@
+package com.socialmedia.controllers;
+
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+@RestController
+@RequestMapping(value = "/users")
```

Comment:
please change mapping path to format "api/v1(optional)/${entities}". It will give an opportunity to distinguish server rest api url from frontend routes